index.bs

```diff
@@ -3,17 +3,13 @@ Group: WHATWG
 H1: Console
 Shortname: console
 Text Macro: TWITTER consolelog
+Text Macro: LATESTRD 2019-12
 Abstract: This specification defines APIs for console debugging facilities.
 Translation: ja https://triple-underscore.github.io/console-ja.html
 Indent: 2
 Markup Shorthands: markdown yes
 </pre>
 
-<pre class="link-defaults">
-spec:infra; type:dfn; text:list
```

Comment:
I would say this is the opposite of centralizing as this would allow us to move the list definition. And also, the moment Infra defines another list you will get this again.